src/services/hasura/hasura.service.ts

```
@@ -281,19 +281,26 @@ export class HasuraService {
-    
-    // First delete all existing records
-    if(arrayOfObjects.length > 0){
-      try {
-        await this.deleteJobs();
-        this.logger.log('Successfully deleted all existing records');
-      } catch (error) {
-        this.logger.error('Error deleting existing records:', error);
-        throw error;
-      }
-    }
+  async insertCacheData(arrayOfObjects: any[], bpps: string[]) {
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)